blocks/carousel/carousel.css

```diff
@@ -134,9 +134,52 @@ main .section.carousel-container {
     background-color: var(--pumpkin-orange-color);
 }
 
+.carousel.block .bouncing-arrow {
+    animation-name: bounce;
+    animation-duration: 0.6s;
+    animation-iteration-count: infinite;
+    display: block;
+    position: fixed;
+    bottom: 30px;
+    left: 50%;
```

Comment:
Can you change this to
`left: calc(50% - 12px);` ?
12px is 1/2 of the SVG width, so adding this lines the arrow up in the center better on our site.